package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vela-ui",
+  "name": "vela-web-ui",
```

Comment:
any particular reasoning for the name change?

Reply:
yes, there is another package called `vela-ui` that results in this being shown at the top of our repo:

![wat](https://user-images.githubusercontent.com/1301201/74753209-b03b3200-5267-11ea-8fb1-fb600addf312.png)